samples/README.md

```diff
@@ -1,3 +1,35 @@
 The sample pipelines give you a quick start to building and deploying machine learning pipelines with Kubeflow.
 * Follow the guide to [deploy the Kubeflow pipelines service](https://www.kubeflow.org/docs/guides/pipelines/deploy-pipelines-service/).
 * Build and deploy your pipeline [using the provided samples](https://www.kubeflow.org/docs/guides/pipelines/pipelines-samples/).
+
+
+
+
+This page tells you how to use the _basic_ sample pipelines contained in the repo.
+
+## Compile the pipeline specification
+
+Follow the guide to [building a pipeline](https://www.kubeflow.org/docs/guides/pipelines/build-pipeline/) to install the Kubeflow Pipelines SDK and compile the sample Python into a workflow specification. The specification takes the form of a YAML file compressed into a `.tar.gz` file. 
```

Comment:
I'm not sure we need to mention YAML.
We should probably use `.zip` instead of `.tar.gz`.
```suggestion
Follow the guide to [building a pipeline](https://www.kubeflow.org/docs/guides/pipelines/build-pipeline/) to install the Kubeflow Pipelines SDK and compile the sample pipeline written in Python to pipeline package.
```

Reply:
This was copied from the previous README in the basic directory. If you do not mind, I will address the comments in the another PR.

Reply:
Just to confirm, will there be a guideline available here for contributors to add samples as well as associated tests?

Reply:
SG.